src/vorta/assets/UI/archivetab.ui

```diff
@@ -28,11 +28,6 @@
        <bold>false</bold>
       </font>
      </property>
-     <property name="styleSheet">
-      <string notr="true">QToolBox::tab {
```

Comment:
Could elaborate on the style issue?
I've also noticed that quite a lot of bold texts on my Qt apps look broken since I've updated my OS.
I've originally thought that might be a flatpak problem.
Is this a known Qt bug?

Reply:
I think a Qt bug. I just made it bold, but it totally destroyed the style of that widget.

Reply:
So better be careful with any styles.

Reply:
Good to know!